env/mysql/knobs.py

```diff
@@ -1,9 +1,83 @@
-class Knobs(object):
+from ..template import Knob
+
+
+class MySQLKnobs(object):
     def __init__(self):
-        pass
+        # TODO: fetch default value from mysql
+        self.knobs_attrs = [
+            {
+                "name": 'table_open_cache',
+                "range": [1, 10240],
+                "default": 512,
+                "type": "int"
+            },
+            {
+                "name": 'innodb_buffer_pool_size',
+                "range": [1048576, 34359738368],
+                "default": 34359738368,
+                "type": "int"
+            },
+            {
+                "name": 'innodb_buffer_pool_instances',
+                "range": [1, 64],
+                "default": 8,
+                "type": "int"
+            },
+            {
+                "name": 'innodb_purge_threads',
+                "range": [1, 32],
+                "default": 1,
+                "type": "int"
+            },
+            {
+                "name": 'innodb_read_io_threads',
+                "range": [1, 64],
+                "default": 12,
+                "type": "int"
+            },
+            {
+                "name": 'innodb_write_io_threads',
+                "range": [1, 64],
+                "default": 12,
+                "type": "int"
+            },
+        ]
+        self.names = list(map(lambda x: x["name"], self.knobs_attrs))
```

Comment:
Whether the order of name remain same between different execution of this statement? Maybe we can sort it first?